### PR TITLE
 [zh] sync docs  content/zh/docs/ops/best-practices/security/index.md

### PR DESCRIPTION
**Pull request by gssjl2008**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/concepts/traffic-management/index.md [hunk 1]
```diff
@@ -156,31 +156,23 @@ destination 片段还指定了 Kubernetes 服务的子集，将符合此规则
 apiVersion: networking.istio.io/v1alpha3
 kind: VirtualService
 metadata:
-  name: bookinfo
+name: bookinfo
```
Comment (windsonsea) [inline content/zh/docs/concepts/traffic-management/index.md:159]: 原文是对的，请与 en 保持一致

File: content/zh/docs/concepts/traffic-management/index.md [hunk 2]
```diff
@@ -156,31 +156,23 @@ destination 片段还指定了 Kubernetes 服务的子集，将符合此规则
...
 spec:
-  hosts:
-    - bookinfo.com
-  http:
-  - match:
+hosts:
+- bookinfo.com
+http:
+- match:
```
Comment (windsonsea) [inline content/zh/docs/concepts/traffic-management/index.md:164]: 这些代码需要跟 en 保持一致

**Review by windsonsea — COMMENTED** [review]
如果 zh 中的代码有问题，需要先修改 en source。

等 en source 修改并 merge 后，才需要修改 zh 版本

File: content/zh/docs/concepts/traffic-management/index.md (hunk 2 above)
Comment (gssjl2008) [reply content/zh/docs/concepts/traffic-management/index.md]: 已经修改完毕

**Review by windsonsea — APPROVED** [review]
LGTM

**Pull request merged**